testcases/single_cell_ispol/job_script.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
The test cases dont have batch scripts associated with them, especially ones with person specific info. Remove from repo.